src/Nethermind/Nethermind.Optimism/Rpc/OptimismPayloadAttributes.cs

```diff
@@ -99,6 +114,18 @@ public override PayloadAttributesValidationResult Validate(ISpecProvider specPro
             return PayloadAttributesValidationResult.InvalidPayloadAttributes;
         }
 
+        IReleaseSpec releaseSpec = specProvider.GetSpec(ForkActivation.TimestampOnly(Timestamp));
+        if (!releaseSpec.IsOpHoloceneEnabled && EIP1559Params is not null)
+        {
+            error = $"{nameof(EIP1559Params)} should be null before Holocene";
+            return PayloadAttributesValidationResult.InvalidPayloadAttributes;
+        }
+        if (releaseSpec.IsOpHoloceneEnabled && EIP1559Params?.Length != 8)
```

Comment:
Add some const like:

```suggestion
        if (releaseSpec.IsOpHoloceneEnabled && EIP1559Params?.Length != OptimismEIP1559Parameters.Length)
```